docs/assignment2.md

```diff
@@ -1 +1,157 @@
+player to reach the goal.
+
+## State machine diagrams									
+Author(s): Elizabeth, Gemma
+
+#### Player State Machine
+
+The player state machine represents the Player class, and how the Player class interacts with the Action class to complete the prompts from the command line. The diagram, entitled “Player State Machine”, describes the possible actions that a player can execute in order to change their state. For the scope of this assignment, this includes interacting with items and locations. 
+ 
+ ![Player State Machine](https://github.com/sarahpazik/software-design-vu/blob/Assignment2/Player%20State%20Machine.png)
+ 
+It is important to note that all of the features in blue on the state machine diagram are currently prescriptive and not yet implemented. 
+	
+The state machine in the diagram describes the possible states a player can adopt as they play the game. The game begins with no player. The user is prompted to provide the player with a name, and then this player is initialized using the corresponding name and JSON file. This is represented by the first arrow in the diagram, labeled with initPlayer(name), and then the following state. Upon entry into this state, the “Initial Player” begins in the “start room” specified in the JSON file, with no objects in their inventory. The player then has two options for the next commands they want to execute, i.e. the next states they can adopt: moving to a new location (“move”), or picking up an item (“pickUp”). 
+
+If the player chooses to enter a new location, they move to the “Player in New Location” state in the diagram. Their current location is changed to the new one, and then displayed while they are in that state. If they choose to pick up an item in whichever location they are currently located, they move to the “Player with Item” state. That item is added to the player’s inventory, and stored there while they are in this state. The player can then choose if they want to move rooms again, or put down the item. The player can only put down an item if they are holding it (a feature which is currently only prescriptive). If they choose to put down this item, they exit the state in which they have that item, moving back to a state in which they are located in their current location but without the item in their inventory.
+
+This continues in a cyclical way, with the player being able to move and pick up or put down items as they please. This cycle terminates when they enter the “end room” specified in the JSON file. Once they enter this room, their end time will be displayed (currently a prescriptive feature), as well as a win or lose message depending on whether or not they completed their objective within the bounds of the time limit. Entering the final state means that either the player has won or lost the game, which is why the diagram displays two alternate transitions following the end room, i.e. the two possible final states for the player. 
+
```

Comment:
-Description: Ok
-Model:
1) Your notation is lacking. noPlayer=true is what: a state name or an entry activity?
2)initPlayer(name) is an event, where is the guard? i.e checking if a valid name is entered. Where is the effect? i.e creating a player.
3)All your other transitions are just the effects that take place from calls to operations in your class diagram, but you don't actually show the transition events and the possible guards.
4)The state Initial Player would make more sense as Idle Player since from what you're portraying if a player with item puts down said item, they become an initial player again.
5)The lose event seems to me that it should terminate your state machine and therefore be followed by a terminate node.
6)After Player in Final Location, you modeled a decision node that does not have a clear decision event. It also lacks a clear guard for which the timeLimit is evaluated.
7)With regards to Player with Item and Player in new Location. Those states seem logical to not be mutually exclusive. Once a player with item moves to a new location, shouldn't they still be a player with item? These would be concurrent substates of a grander composite state.

Most elements in your description are semantically correct but a lot of that was lost in the translation to a state machine diagram.